### PR TITLE
UT migration, deprecating InstrumentationTestCase

### PR DESCRIPTION
**Pull request by piqiums**
This is the update on all subclass of InstrumentationTestCase as part of https://identitydivision.visualstudio.com/DevEx/Android/_workitems?id=264686 .

This PR touched 19 files. I've ran them all before sending out the PR, all test passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -292,7 +292,7 @@ mContext.acquireTokenSilent(String resource, String clientId, String userId, fin
 	* If target version is lower than 23, calling app has to have the following permissions declared in manifest(http://developer.android.com/reference/android/accounts/AccountManager.html):
 		* GET_ACCOUNTS
 		* USE_CREDENTIALS
-		* MANAGE_ACOUNTS
+		* MANAGE_ACCOUNTS
```
Comment (addev-ashish) [inline README.md]: As much as i like this, no unrelated changes in a PR, you can make this change when you fix #898

Comment (piqiums) [inline README.md]: That's a change already checked into 'dev' branch by the author. It appeared in the "New changes since you last viewed" section in this PR, but it's not part of my change. It won't affect my PR.

Comment (addev-ashish) [inline README.md]: That means you need to merge dev branch into your branch. Please do that and update the PR.
